src/app/containers/create-movie/index.jsx

```diff
@@ -1,28 +1,36 @@
-'use client'
+import { useMutation } from "@tanstack/react-query";
+import axios from "axios";
+import MovieForm from "./movieForm";
 
-import { useMutation } from "@tanstack/react-query"
-import axios from "axios"
-
-const createMovie = async () => {
-    const response = await axios.post('/api/movies', {id: 7, name: "Dune" })
-
-    return response.data
-}
+const createMovie = async (movieData) => {
+  const response = await axios.post("/api/movies", movieData);
+  return response.data;
+};
 
 export const CreateMovie = () => {
-    const { mutate, data, isSuccess, isError, error } = useMutation({
-        mutationFn: createMovie, 
-        mutationKey: ["create-movie"]
-    })
+  const { mutate, data, isSuccess, isError, error } = useMutation({
+    mutationFn: createMovie,
+    mutationKey: ["create-movie"],
+  });
 
-    return <>
-        {isError && <div className="text-red-600">{error.message}</div>}
-        <button onClick={() => mutate()}>Create movie</button>
-        {isSuccess && <div>
-            Newly created movie
-            <p>This is id: {data.id}</p>
-            <p>This is movie name: {data.name}</p>
-            </div>}
-    </>
+  function handleMovieSubmit(data) {
```

Comment:
This function is just a renaming of the mutate function. This could be done where we have the useMutation like this:

`const { mutate: handleMovieSubmit, data, isSuccess, isError, error } = useMutation({})`

Reply:
renamed the mutate func